ui/components/logs/data-table.tsx

```
@@ -0,0 +1,453 @@
+
+  const updatePagination = (updates: Partial<Pagination>) => {
+    onPaginationChange({ ...pagination, ...updates });
+  };
+
+  const goToPage = (page: number) => {
+    const newOffset = (page - 1) * pagination.limit;
+    updatePagination({ offset: newOffset });
+  };
+
+  const handlePageSizeChange = (newSize: string) => {
+    const newLimit = parseInt(newSize);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)